features/map.go

```diff
@@ -227,7 +227,7 @@ func init() {
 }
 
 // MapFlags document which flags may be feature probed.
-type MapFlags = sys.MapFlags
```

Comment:
This is the breaking change the linter is complaining about. I don't think this should make a difference in practice.

Reply:
Nope, should be fine since external callers never had access to `sys.MapFlags` anyway.